pkg/provision/partial.go

```diff
@@ -0,0 +1,64 @@
+package provision
+
+import (
+	"context"
+
+	"github.com/cortezaproject/corteza-server/pkg/filter"
+	"github.com/cortezaproject/corteza-server/pkg/rbac"
+	"github.com/cortezaproject/corteza-server/store"
+	"github.com/cortezaproject/corteza-server/system/types"
+	"go.uber.org/zap"
+)
+
+type (
+	uConfigFn func(context.Context, store.Storer, *zap.Logger) bool
+	uConfig   struct {
+		dir string
+		fn  uConfigFn
+	}
+)
+
+// provisionPartialAuthClients checks for a specific set of auth client rbac rules
+func provisionPartialAuthClients(ctx context.Context, s store.Storer, log *zap.Logger) bool {
```

Comment:
I don't understand the logic here?
All you need to check if there is at least one rule for `system:auth-client:*`.
If there is, assume provision was done and don't run it again (for that part)

Reply:
True, fixed.